pkg/services/navtree/navtreeimpl/applinks.go

```diff
@@ -255,8 +255,9 @@ func (s *ServiceImpl) hasAccessToInclude(c *contextmodel.ReqContext, pluginID st
 func (s *ServiceImpl) readNavigationSettings() {
 	s.navigationAppConfig = map[string]NavigationAppConfig{
 		"grafana-k8s-app":                  {SectionID: navtree.NavIDMonitoring, SortWeight: 1, Text: "Kubernetes"},
-		"grafana-synthetic-monitoring-app": {SectionID: navtree.NavIDMonitoring, SortWeight: 2, Text: "Synthetics"},
+		"grafana-app-observability-app":    {SectionID: navtree.NavIDMonitoring, SortWeight: 2, Text: "Application"},
```

Comment:
Do we confirm we don't annotate by "Preview" like "Application (preview)"?

Reply:
@cyrille-leclerc who can make the decision?

Reply:
@cedricziel or @Rootzreggae ?

Reply:
It resonates with me to add a "(preview)" suffix.

Reply:
added!